### PR TITLE
Add some missing value to augments table, ...

### PR DESCRIPTION
**Pull request by TeoTwawki**
...and fix some mistakes left in modifier.h and status.lua

NULL mistakenly replaced with nullptr in mod names, and dangling mods you ceased using which hopefully are not still left any scripts now that we have a table for those - I **did** do a quick search of the item scripts, didn't get any results.

### CODE REVIEW

File: src/map/modifier.h [hunk 1]
```diff
@@ -526,18 +514,18 @@ enum MODIFIER
...
 
-    //Elemental nullptr Chance
-    MOD_FIRE_nullptr                 = 0x1D3, // (modId = 467)
-    MOD_EARTH_nullptr                = 0x1D4, // (modId = 468)
-    MOD_WATER_nullptr                = 0x1D5, // (modId = 469)
-    MOD_WIND_nullptr                 = 0x1D6, // (modId = 470)
-    MOD_ICE_nullptr                  = 0x1D7, // (modId = 471)
-    MOD_LTNG_nullptr                 = 0x1D8, // (modId = 472)
-    MOD_LIGHT_nullptr                = 0x1D9, // (modId = 473)
-    MOD_DARK_nullptr                 = 0x1DA, // (modId = 474)
```
Comment (xdemolish) [inline src/map/modifier.h]: @teschnei r the best

**Comment by teschnei** [comment]
too bad visual studios replace all symbol isn't until 2015.. Could have
avoided all these nullptr things. Oh well, changing the name of an enum
everywhere isn't gonna break anything

**Comment by maxtherabbit** [comment]
I'm using a table for the null chance in battleutils::MagicDmgTaken so they need to be fixed there as well

**Comment by teschnei** [comment]
surely he attempted do compile after changing enum names. surely!

**Comment by TeoTwawki** [comment]
Surely I would compile after putting back the obviously correct stuff! No not really. I have to much to do to waste my time with the trivial. But I will go fix now that its pointed out, and make sure it works.

**Comment by TeoTwawki** [comment]
"Build succeeded" so there shouldn't be any more to find.

**Comment by maxtherabbit** [comment]
Thanks bro

**Comment by teschnei** [comment]
I mean, if this wasn't an enum defined in a header then I'd agree with you.
But its an enum defined in a header that currently compiled, and all it
takes is 2 seconds to hit f7 to make sure nothing else referenced them.

**Comment by TeoTwawki** [comment]
Its done now anyway, but I see this as code review doing what its supposed to do. No offense meant, I'm just not that good a coder to think of those things as 100% of my learning has come from searching and asking questions - no schooling on any language at all, so the important of it being an enum in a header and thinking beyond "is thing I did **here** correct" is lost on me until things like this happen.

**Comment by TeoTwawki** [comment]
tl;dr
I was being lazy, I was being ignortant.

**Comment by TeoTwawki** [comment]
tl;dr
I wasn't being lazy, I was being ignortant.

**Comment by teschnei** [comment]
I think it's still a good idea to compile if you changed any core files.
Don't be like me, I'm a bad influence

**Comment by TeoTwawki** [comment]
I'll remember to do at least a build if I even touch them from now on.

**Comment by maxtherabbit** [comment]
why'd you take out the pet mods from status.lus?

**Comment by teschnei** [comment]
Because I forgot to in my commit

**Comment by maxtherabbit** [comment]
so it's straight to merge now?
